examples/applications/lt-consumer/src/main.rs

```diff
@@ -225,7 +225,7 @@ async fn evaluate_docker_stats(
                 let cpu_delta =
                     cpu_stats.cpu_usage.total_usage - stats.precpu_stats.cpu_usage.total_usage;
 
-                if number_cpus == None {
+                if number_cpus.is_none() {
```

Comment:
I'm guessing Clippy complained about this?

Reply:
Exactly, though I did not investigate why it did not complain before.